README.md

```diff
@@ -1,2 +1,149 @@
+
+The time window against which events in the active window are compared to test for regressions. For using the Increasing Error Gate, a baseline time window is required
+
+* Example: 14d would be a two week baseline time window.
+
+## Event Volume Threshold
+
+The minimal number of times an event of a non-critical type (e.g. uncaught) must take place to be considered severe.
+
+* If a New event has a count greater than the set value, it will be evaluated as severe and could break the build if its event rate is above the Event Rate Threshold.
+* If an Existing event has a count greater than the set value, it will be evaluated as severe and could break the build if its event rate is above the Event Rate Threshold and the Critical Regression Threshold.
+* If any event has a count less than the set value, it will not be evaluated as severe and will not break the build.
+
+## Event Rate Threshold (0-1)
+
+The minimum rate at which event of a non-critical type (e.g. uncaught) must take place to be considered severe. A rate of 0.1 means the events is allowed to take place <= 10% of the time.
+
+* If a New event has a rate greater than the set value, it will be evaluated as severe and could break the build if its event volume is above the Event Volume Threshold.
+* If an Existing event has a rate greater than the set value, it will be evaluated as severe and could break the build if its event volume is above the Event Volume Threshold and the Critical Regression Threshold.
+* If an event has a rate less than the set value, it will not be evaluated as severe and will not break the build.
+
+## Regression Delta (0-1)
+
+The change in percentage between an event's rate in the active time span compared to the baseline to be considered a regression. The active time span is the Active Time Window or the Deployment Name (whichever is populated). A rate of 0.1 means the events is allowed to take place <= 10% of the time.
+
+* If an Existing event has an error rate delta (active window compared to baseline) greater than the set value, it will be marked as a regression, but will not break the build.
+
+## Critical Regression Threshold (0-1)
+
+The change in percentage between an event's rate in the active time span compared to the baseline to be considered a critical regression. The active time span is the Active Time Window or the Deployment Name (whichever is populated). A rate of 0.1 means the events is allowed to take place <= 10% of the time.
+
+* If an Existing event has an error rate delta (active window compared to baseline) greater than the set value, it will be marked as a severe regression and will break the build.
+
+## Apply Seasonality
+
+If peaks have been seen in baseline window, then this would be considered normal and not a regression. Should the plugin identify an equal or matching peak in the baseline time window, or two peaks of greater than 50% of the volume seen in the active window, the event will not be marked as a regression.
```

Comment:
Add:

## Debug Mode

If checked, all queries and results will be displayed in the OverOps reliability report. For debugging purposes only.